docs/qsimcirq_tutorial.ipynb

```diff
@@ -0,0 +1,594 @@
+          "text": [
+            "Cirq results:\n",
+            "[0.70710677+0.j 0.        +0.j]\n",
+            "\n",
+            "qsim results:\n",
+            "[(0.7071067690849304+0j), 0j]\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "zy5XRdkm219l",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Performance benchmark\n",
+        "\n",
+        "The code below generates a depth-16 circuit on a 4x5 qubit grid, then runs it against the basic Cirq simulator. For a circuit of this size, the difference in runtime can be significant - try it out!"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "SyRpm08R3qCy",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 69
+        },
+        "outputId": "790ad8b5-e665-44e4-c33c-177cf98dba5a"
+      },
+      "source": [
+        "import time\n",
+        "\n",
+        "qubits = [cirq.GridQubit(i,j) for i in range(4) for j in range(5)]\n",
+        "\n",
+        "# Generates a random circuit on the provided qubits.\n",
+        "circuit = cirq.experiments.random_rotations_between_grid_interaction_layers_circuit(\n",
+        "    qubits=qubits, depth=16)\n",
+        "\n",
+        "# Simulate the circuit with Cirq and print the runtime.\n",
+        "cirq_simulator = cirq.Simulator()\n",
+        "cirq_start = time.time()\n",
+        "cirq_results = cirq_simulator.simulate(circuit)\n",
+        "cirq_elapsed = time.time() - cirq_start\n",
+        "print(f'Cirq runtime: {cirq_elapsed}')\n",
+        "print()\n",
+        "\n",
+        "# Simulate the circuit with qsim and print the runtime.\n",
+        "qsim_simulator = qsimcirq.QSimSimulator()\n",
+        "qsim_start = time.time()\n",
+        "qsim_results = qsim_simulator.simulate(circuit)\n",
+        "qsim_elapsed = time.time() - qsim_start\n",
+        "print(f'qsim runtime: {qsim_elapsed}')\n"
+      ],
+      "execution_count": 9,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "Cirq runtime: 6.024493217468262\n",
+            "\n",
+            "qsim runtime: 0.12877583503723145\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LgWOEMLKutsq",
+        "colab_type": "text"
+      },
+      "source": [
+        "# Distributed execution\n",
+        "\n",
+        "qsimh (qsim-hybrid) is a second library in the qsim repository that takes a slightly different approach to circuit simulation. When simulating a quantum circuit, it's possible to simplify the execution by assigning fixed values to certain gates. This operation is called \"slicing\" (or \"cutting\") the gates, because it removes a portion of the overall state.\n",
```

Comment:
"assigning values to gates" might be a bit oversimplifying (and confusing). It might be better to say about the Schmidt decomposition, etc. Perhaps that depends on the target audience of this tutorial.

Reply:
This is mainly for users, not developers, so I don't think we need to go into too much detail here. You're right about oversimplifying, though - made the terminology a bit more accurate.